### PR TITLE
Consul Enterprise docs update

### PR DESCRIPTION
**Pull request by BeenzSyed**
Updated the top portion of the Overview for Consul Enterprise.

Added a page for Audit Logging, linking it to the sidebar and on the Consul Enterprise page.

### CODE REVIEW

File: website/pages/docs/enterprise/audit-logging.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+page_title: Consul Enterprise Audit Logging
+sidebar_title: Audit Logging
+description: >-
+  Consul Enterprise provides the ability to write events of user behavior with Consul’s API so operations and security users can perform legal compliance auditing.
+---
+
+# Audit Logging
+
+<EnterpriseAlert>
+  This feature is available in all versions of <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>.
```
Comment (rboyer) [inline website/pages/docs/enterprise/audit-logging.mdx:12]: Shouldn't this mention 1.8+?

File: website/pages/docs/enterprise/audit-logging.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</EnterpriseAlert>
+
+Audit logging captures a clear and actionable log of authenticated events (both attempted and committed) that Consul processes and compiles them into a JSON format for easy export. These events contain a timestamp, the operation performed, and the user who initiated the action.
+
+Audit logging provides greater insight and accountability for security teams that are concerned with automating service networking tasks.
```
Comment (preetapan) [inline website/pages/docs/enterprise/audit-logging.mdx:17]: Rewording suggestion "Audit logging enables security and compliance teams within an organization to get greater insight into Consul access and usage patterns"

File: website/pages/docs/enterprise/audit-logging.mdx (hunk 1 above)
Comment (BeenzSyed) [reply website/pages/docs/enterprise/audit-logging.mdx:12]: Yes!

I see in the other Enterprise feature pages, in the EnterpriseAlert area, it's written that the feature is available in a particular module. Ex. Namespaces requires the Policy and Governance module (https://www.consul.io/docs/enterprise/namespaces). I see from the pricing page (https://www.hashicorp.com/products/consul/pricing/), Audit Logging falls under the same category. To stay consistent, I can reword to: 'This feature requires Consul Enterprise with the Governance and Policy module.'

I can also add 'Consul Enterprise v1.8.0 adds Audit Logging...' to the first paragraph.

File: website/pages/docs/enterprise/audit-logging.mdx (hunk 2 above)
Comment (BeenzSyed) [reply website/pages/docs/enterprise/audit-logging.mdx:17]: Thanks - I'll make that change.

**Pull request merged**